packages/databyss-ui/primitives/Control/TextControl.js

```diff
@@ -50,6 +51,18 @@ const TextControl = ({
     },
     zIndex: active ? 2 : 'unset',
   }
+
+  useEffect(
```

Comment:
I think the "autofocus" feature should be moved to the `SourceModal` component, it can introduce weird side effects at this level of the render tree